camera_utils.cpp

```diff
@@ -0,0 +1,77 @@
+#include "camera_utils.hpp"
+
+#include <interfaces/image_format.hpp>
+
+#include <iostream>
+
+namespace bnb
+{
+
+    void camera_utils::full_image_to_pixel_buffer(bnb::full_image_t &image, pixel_buffer_callback_t callback)
+    {
+        int32_t width = image.get_format().width;
+        int32_t height = image.get_format().height;
+        int32_t stride = image.get_format().width;
+
+        if (image.has_data<bnb::yuv_image_t>()) {
+            auto yuv_image = image.get_data<bnb::yuv_image_t>();
+            auto yuv_format = yuv_image.get_yuv_format();
+
+            bnb::oep::interfaces::image_format outfmt;
+            uint8_t tocase =
+                (static_cast<uint8_t>(yuv_format.format == bnb::yuv_format::yuv_nv12) << 2) |
+                (static_cast<uint8_t>(yuv_format.standard == bnb::color_std::bt601) << 1) |
+                (static_cast<uint8_t>(yuv_format.range == bnb::color_range::full) << 0);
```

Comment:
this is a trick, it is ok for now, because either of these variables have only 2 values, but it is not extendable.
I'm fine with that, no any refactoring is required now.

Just I think that it is worth to add some comment about used "encoding" method, i.e. which bit means what (for example if bit 2 is YUV format, possible values are nv12 (bit set) and i420 (bit not set)), because from "encoding" statement it is not clear what other value is (what value is used when bit is not set)
Also it is worth to add comment that this trick should replaced with something other in case of adding new values to mentioned enums (for example new YUV format or color standard). this is very unlikely to happen soon, but such trick in code may lead to unexpected bugs which will require debugging